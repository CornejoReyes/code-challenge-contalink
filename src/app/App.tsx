import { config } from '@gluestack-ui/config';
import { GluestackUIProvider, Text } from '@gluestack-ui/themed';

function App() {
  return (
    <GluestackUIProvider config={config}>
      <Text>Hello There</Text>
    </GluestackUIProvider>
  );
}

export default App;
