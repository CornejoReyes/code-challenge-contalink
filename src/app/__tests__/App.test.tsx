import { render, screen } from '@testing-library/react-native';
import App from '../App';

describe('App component', () => {
  it('should render correctly', async () => {
    render(<App />);
    const greeting = await screen.findByText('Hello There');

    expect(screen.toJSON()).toMatchSnapshot();
    expect(greeting).toBeOnTheScreen();
  });
});
