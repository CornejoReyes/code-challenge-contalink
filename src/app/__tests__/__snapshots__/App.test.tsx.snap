// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly 1`] = `
<Text
  data-style=""
  dataSet={
    {
      "style": "",
    }
  }
  size="md"
  style={
    [
      undefined,
      {},
      undefined,
      {
        "color": "#525252",
        "fontFamily": undefined,
        "fontStyle": "normal",
        "fontWeight": "400",
        "letterSpacing": 0,
      },
      undefined,
      {},
      undefined,
      {
        "fontSize": 16,
      },
      undefined,
      undefined,
    ]
  }
>
  Hello There
</Text>
`;
